phoenix-protocol/src/main/MetaDataService.proto

```diff
@@ -72,6 +79,12 @@ message GetFunctionsRequest {
   optional int32 clientVersion = 5;
 }
 
+message GetSchemaRequest {
+  required string schemaName = 1;
+  required int64 clientTimestamp = 2;
+  optional int32 clientVersion = 3;
```

Comment:
Should the client version be optional or required? Considering this is a new metadata entity, I would think most of the things here should have "required" unless of course functionally they don't have to be present always. This applies to all the other schema related requests.